src/constants.js

```diff
@@ -104,8 +104,8 @@ export const NavigationModeTypes = {
   ARROW_KEYS: 1,
 };
 
-export const RowHeight = 36;
-export const WindowHeight = 300;
+export const ROW_HEIGHT = 36;
```

Comment:
these constant names are pretty generic and don't really fit in for the context of the global constants file. maybe we can update these to:

```
EMOTE_MENU_GRID_ROW_HEIGHT = 36;
EMOTE_MENU_GRID_HEIGHT = 300;
EMOTE_MENU_SIDEBAR_ROW_HEIGHT = 48;
```